packages/web3-constants/evm/mask-box.json

```diff
@@ -35,6 +38,9 @@
         "Arbitrum": 0,
         "Arbitrum_Rinkeby": 0,
         "xDai": 0,
+        "Optimism": 2283229,
```

Comment:
@UncleBill Please check this.

Reply:
@developerfred It should be **1272779** instead. https://optimistic.etherscan.io/tx/0xa5fa95b9c2305632ff0f10fed5e1277461f07e49855c2e6b5913279f146b00f5

Please check it from https://github.com/DimensionDev/mysteryBox/#mysterybox--masktestnft.